ctapipe/ml/tools/apply_energy_regressor.py

```diff
@@ -83,6 +93,31 @@ def start(self):
                 mode="a",
                 overwrite=self.overwrite,
             )
+        self.loader.load_instrument = False
+        # TODO: Might need to turn this on if we want to use dl1 features as weights
+        self.loader.load_dl1_parameters = False
```

Comment:
This already happens with `weight=konrad`, doesn't it? Add a test for `weight="konrad"`?